dcrtimed/dcrtimed.go

```diff
@@ -1319,7 +1319,8 @@ func _main() error {
 			loadedCfg.WalletCert,
 			loadedCfg.WalletHost,
 			loadedCfg.EnableCollections,
-			[]byte(loadedCfg.WalletPassphrase))
+			[]byte(loadedCfg.WalletPassphrase),
+			fsbeLog, walletLog)
```

Comment:
The logging vars are package level global variables  You don't need to pass it to this initialization function.  Just do:
```
filesystem.UseLogger(fsbeLog)
```

Reply:
👍